src/hooks/v2/transactor/PayETHPaymentTerminal.ts

```diff
@@ -17,7 +17,11 @@ type PayV2ProjectTx = TransactorInstance<{
   value: BigNumber
 }>
 
-export function usePayETHPaymentTerminalTx(): PayV2ProjectTx {
+export function usePayETHPaymentTerminalTx({
```

Comment:
This file diff is an example of how we'll add transactionTitles to every TransactorInstance

Titles will default to the contract function name (see transactor hook) if we don't define them